app/views/media/index.html.erb

```diff
@@ -0,0 +1,37 @@
+<div class="row">
+  <div class="col-lg-12">
+    <table class="table table-hover">
+      <tr>
+        <th>
+          Ranking
+        </th>
```

Comment:
For readability I would consolidate each of these to look like:

`<th>Ranking</th>`
`<th>Name</th>`
`<th>Upvote</th>`

etc.